client/components/Reviews/styles/ReviewsList.styled.js

```diff
@@ -1,8 +1,10 @@
 import styled from 'styled-components';
 
 export const List = styled.div`
```

Comment:
Consider a default export instead of an in-line export for style-files that contain only one exported styled-component.

Reply:
I was thinking of adding this once I was 100% done with modifying styling, but this is a good point & not a hard fix - will update to fix next go.